mappings/net/minecraft/datafixer/fix/EntityUuidFix.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_4960 net/minecraft/datafixer/fix/EntityUuidFix
+	FIELD field_23122 RIDEABLE_TAMEABLES Ljava/util/Set;
+	FIELD field_23123 TAMEABLE_PETS Ljava/util/Set;
+	FIELD field_23124 BREEDABLES Ljava/util/Set;
+	FIELD field_23125 LEASHABLES Ljava/util/Set;
+	FIELD field_23126 OTHER_LIVINGS Ljava/util/Set;
+	FIELD field_23127 PROJECTILES Ljava/util/Set;
+	METHOD <init> (Lcom/mojang/datafixers/schemas/Schema;)V
+		ARG 1 outputSchema
+	METHOD method_26022 updateAngryAtMemory (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 dynamic
+	METHOD method_26024 updateEvokerFangs (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26026 updateZombieVillager (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26028 updateAreaEffectCloud (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26030 updateShulkerBullet (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26031 updateItemEntity (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26032 updateFox (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26033 updateZombifiedPiglin (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26034 updateTameable (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26035 updateBreedable (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26036 updateLeashable (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
+		ARG 1 root
+	METHOD method_26037 updateLiving (Lcom/mojang/datafixers/Dynamic;)Lcom/mojang/datafixers/Dynamic;
```

Comment:
`updateLivingAttributes` or `updateEntityAttributes` maybe? just `updateLiving` seems to be quite broad in it's targeting

Reply:
update living because this applies to all living (called by `updateLeashable` as well, which is called by `updateBreedable`)